lib/runner/cli/cli.js

```diff
@@ -114,6 +115,33 @@ class CliRunner {
       }
     }
 
+    const {desiredCapabilities} = this.test_settings;
+
+    if (this.argv.parallel && (isSimulator(desiredCapabilities) || isRealIos(desiredCapabilities) || isAndroid(desiredCapabilities))) {
```

Comment:
`isSimulator(desiredCapabilities) || isRealIos(desiredCapabilities) || isAndroid(desiredCapabilities)`

Can't these be wrapped in a single helper function?

Reply:
Fixed in [d02f08c](https://github.com/nightwatchjs/nightwatch/pull/3443/commits/d02f08c9af0a362c1f4cbd4951cd51659888f750)